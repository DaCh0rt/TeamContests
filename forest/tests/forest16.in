530 758
-3941 -9171
-9982 -7733
7022 -5212
-9590 -8414
-7626 -8368
5330 7942
-8611 -7396
2970 -7184
5858 -9158
-8856 -5938
-5240 2623
-9851 1732
3422 3144
-7501 4110
6407 6758
4509 -4811
9313 6995
-838 2355
7109 938
659 -840
3757 -7505
1624 8536
5989 -708
5588 3898
6867 6589
-670 9576
-4044 -3564
-594 1637
-3246 -2026
992 5267
3274 4073
6981 -1300
-6633 -310
4122 -334
-4306 -7153
3518 -7982
-7407 7573
-3454 -3865
-7442 8506
5399 -9588
-3355 -1674
5918 3793
-4056 4381
8361 -1909
-4740 3206
9264 8050
-6891 4778
-9285 4129
4642 -1856
7434 -7069
-53 -5854
-4377 -6538
2141 9996
2010 -5471
3079 -4561
1241 -3907
9686 -5528
3072 6282
-1810 -7469
5604 3556
-8278 -5403
-8041 -6555
1226 833
-5339 -9852
-7892 -5028
-5380 8550
-2641 1775
6499 -4635
46 -5741
-6965 6308
3329 -5042
-9138 -9250
-2466 2216
-9706 -9595
8495 -2501
9234 1507
4583 2821
7217 8278
3566 2678
429 -9338
-1490 9289
-4394 5887
-1669 4382
-3507 3818
2663 5216
4007 6247
-2057 5762
-7067 4006
7304 -3710
6313 5336
4354 8935
1656 432
2720 3675
9678 4133
-8364 -3216
-514 -9093
958 -1595
2835 6487
397 -5617
-8283 5323
9032 -1112
3976 4471
46 6493
-2461 -2254
-4912 2957
1756 -7117
1929 -232
-6899 -3208
6031 9961
-7677 3101
7299 -3366
7538 -9201
9985 6163
-9134 4518
1505 -9024
193 4212
-1714 -71
-724 8976
7854 -842
3635 -8518
4000 -2445
222 -7092
-2454 -2881
-8339 -4637
8264 3610
8736 -4538
-8185 2751
677 -6868
29 -7785
-2869 -3696
9907 -5966
-8160 -8479
-6869 -577
-6030 9811
8355 1918
585 5485
-8100 7117
1724 -2465
-7787 5105
2282 1232
-9513 -382
6820 -8762
-952 738
-5910 -7635
3024 -3665
2701 -7324
843 -3181
-5802 -1711
3651 9099
-553 -9221
-9281 766
7047 -702
-1624 -2377
-6017 3105
-1630 8829
-8382 1595
-1234 -3265
-376 5725
-697 -6447
-2261 -3801
-9396 -3554
6164 -9264
-6412 -4408
6545 7665
1271 -3933
-7345 -480
-4181 506
7078 1981
2759 7639
1788 6271
2975 403
-4401 3993
-5237 4932
8518 8815
2317 5381
-2418 5761
54 1980
7523 7203
2337 5432
3938 -9183
9176 -3298
-1932 5091
-5171 -1421
-2058 -3222
-469 2993
-5815 -3858
6582 -2857
8711 6021
-7654 -9409
9745 -2282
143 6302
-9851 8046
7094 -1587
1936 843
-6567 -2471
9038 810
-8360 50
9380 -6533
3836 -2715
1595 5278
9555 4653
3640 -3890
8264 -206
3844 -9334
1404 -6186
-9575 2171
-6560 21
-3402 -8156
7270 6795
9392 3880
9085 -3265
1600 -6277
-8997 7269
4402 -2008
-1991 -9380
-8926 -7803
35 7540
4669 -7765
4030 -7611
-2739 962
2732 5583
4751 -3614
7081 9153
-4318 3601
2042 -3914
7271 -7232
-8184 2305
-97 1106
5813 6998
9331 2
-851 -3053
858 37
-2268 2123
-8648 -7759
-8336 -9964
-9610 6323
2989 6540
1151 -6864
1347 -1659
-5293 1767
7674 6961
2590 5895
-1386 -8262
9348 3836
-2538 -6444
9824 -2330
2151 9755
-3503 2479
6496 -3414
-3828 8338
-5396 -5623
2325 -9105
4526 4523
3320 6788
3744 4874
6095 -6739
-7558 -6612
1427 -7426
5626 -8320
2433 8240
2497 -2090
7917 2048
-687 -2866
-6360 7743
148 4332
-4187 -9609
-1447 9800
9923 8148
-2211 -7366
8739 -823
-5672 4018
304 -6059
-3897 4872
1739 -4623
3070 -6098
-6654 -2149
-8061 -8029
-6800 8700
4564 9054
85 2574
7220 -2315
-4045 7784
4057 4926
1467 2690
-5993 -6045
4779 -5067
6597 4782
-5539 -7311
-8585 1256
8979 -9161
8897 -9494
-4923 -2482
3794 1067
-1362 8040
-9260 -3940
-4342 7342
-7717 4066
-8408 -1754
-7776 9724
-5044 1844
7542 15
-6399 1910
3852 9266
-2714 8299
1251 6248
-6043 -4195
4696 -5707
-3703 4902
-4442 8878
-1363 -1559
629 -1777
-4826 3040
972 -7555
-9717 8681
-4758 -508
-3127 5611
-5052 2947
-7965 -674
2282 3063
8188 -9333
-4778 9861
-8636 -9528
-7633 -3088
-2932 972
-1451 -1958
3240 7769
-6869 4666
-2843 1534
-6946 -8990
-3456 6263
-9171 4936
-5512 6201
-7452 -8683
2178 1996
8143 5688
1789 -1128
5432 5478
1387 4962
-579 -9849
-5646 -5359
-6911 -3765
8013 -7522
-2891 -8038
-1531 -4426
-4880 -1784
-21 5098
-9349 1360
-1233 7794
-3362 -1689
8904 -10
-2475 7745
-1144 -6059
6321 2613
-4029 -6092
-6866 -9656
5503 -927
-5494 -7907
-2267 637
-7021 -8052
5518 3312
-5596 367
7373 -3556
-317 646
6796 -8625
5246 -7093
-228 -3488
-1075 5325
3362 -5339
4189 6416
-8542 -1004
4553 9144
1451 -5978
-9940 7666
680 -7326
4394 -7232
635 7630
-3512 8803
9306 -4953
2358 -2956
-8556 1242
2341 -7857
5455 2684
-8258 -7996
1847 -3340
-6502 1235
-7648 4079
-916 -946
-4946 -237
-5033 -5644
7001 -9305
-191 4549
-9236 -8453
2482 4369
1512 -2682
-3162 -259
2939 9754
-520 2948
2605 8452
97 8071
-4839 809
-6283 -1781
927 2961
181 862
5741 -4160
-3468 4156
-2409 7338
4036 2302
8414 3868
4403 4103
-5546 3795
7675 7813
6930 906
5501 544
640 -7347
-6 2907
-7347 -580
-857 4563
7166 -2660
9644 3480
2063 9270
1676 3657
-9705 -9968
-8677 3254
3139 -8156
-2040 6693
-6234 -1947
-7942 2483
4708 -6829
-1276 -2780
7604 -4586
-7760 8965
-9070 -5516
456 -1293
-715 2006
-4397 -1212
1344 1980
1960 9739
-5304 671
-7654 3416
7575 549
4114 -218
8997 -4043
-9862 -4548
-1152 -3378
2641 1676
1685 -8559
-5606 -9209
5410 -8606
1940 -1353
9485 -3042
-8932 1698
-4167 -3324
5249 4581
-3036 -1178
-9129 6357
4854 8480
2029 7578
-9027 8583
9988 -3889
-2324 -6610
8437 8810
-6251 247
-6626 7059
9853 8359
-8247 1344
8548 6519
-4559 5682
1686 -8358
-510 205
6067 -2720
6129 2567
-3943 -7970
3798 -6437
-1968 -5831
-4756 6215
5996 2096
4998 -4568
-7757 6096
-4219 1978
-8234 -8619
3820 7532
-4115 -7808
-7694 -5686
8091 -896
-3620 -2286
2652 -4545
-8140 9006
7492 7262
-3524 -4441
-1898 9514
354 -3146
-6234 2288
-9769 -8862
-6738 2802
5191 3535
9398 -3751
97 -7193
9410 6586
-324 1927
-4482 -3185
-3456 6060
3270 4106
-7132 -1206
-1249 -1667
-3196 -2120
-2665 -6385
333 5935
-5086 4144
-3391 -1793
-6765 3477
1948 -6570
2494 478
7835 -7767
-4018 3745
-8384 8553
9521 -5622
458 -9632
1438 -1029
-2331 -4049
-3900 7200
4417 -431
-2644 -8296
-2467 -8730
7909 -7393
-9063 -6219
1774 -6136
9015 2352
4816 3240
306 -7611
-3638 5827
