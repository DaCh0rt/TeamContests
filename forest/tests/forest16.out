80230.39 444400950.28
